src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js

```diff
@@ -7,6 +7,7 @@ import 'date-fns';
 import DateFnsUtils from '@date-io/date-fns';
 import { makeStyles } from '@material-ui/core';
 import { MuiPickersUtilsProvider, KeyboardDatePicker } from '@material-ui/pickers';
+import { BasicInputWrapper } from '../.';
```

Comment:
We should probably import only the wrapper component from `../BasicInputWrapper` instead of importing the whole parent folder